### ISSUE TITLE
M> (G)Casper - Infographic

### ISSUE THREAD

**Issue by patrick727** [rchain/bounties#154]
Infographic for

What is Casper? 
Want to know a layout and imagery for Casper infographic. 

see vlads new paper https://github.com/ethereum/research/tree/master/papers/CasperTFG

**Comment by traviagio** [comment]
I did a few infographics on the topics a few years ago. Here are some links: https://cointelegraph.com/news/bitcoin-growth-in-2016-show-us-your-numbers and https://faisalkhan.com/2015/06/30/bitcoin-acceptance-for-everyday-use-infographic/ Also this is another one on a different topic https://medium.com/the-mission/why-working-from-the-beach-in-bali-is-the-new-cool-f84329d3f4aa But it's a hell a lot of work and I don't think I can find the time to do another one. I am happy to collaborate on this if others are interested however.

**Comment by patrick727** [comment]
@traviagio it sure would be amazing to have #180 formed into an infographic. 

also as for the casper infographic, if you could help sketch it out as per, Vlad, Greg and Michaels outline that would also help alot.

What capacity can you work on an infographic?

**Comment by traviagio** [comment]
I usually do the research and build a story line and basic sketching. Then I send it over to a designer to build it and later I would pitch it to major publications. Let me see what I can do.

**Comment by AyAyRon-P** [comment]
Let me know if there is still a need to create this graphic and I can add it to my production schedule. 
@patrick727 @llerner

**Comment by Ojimadu** [comment]
@AyAyRon-P You can add it to your production schedule as no work has been done on it. Try making the inforgraphic an ELI5 type of inforgraphic. I would be willing to assist where necessary.

**Comment by dckc** [comment]
@patrick727 is this still relevant to life as we know it? When is the last time you reviewed **Marketing** issues?

**Comment by pmoorman** [comment]
Closing, in an effort to keep Github organised. 

(feel free to reopen, if needed)
